packages/react-native/src/primitives/TextField/TextField.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useMemo } from 'react';
+import { TextInput, View, ViewStyle } from 'react-native';
+import { Label } from '../Label';
+
+import { styles } from './styles';
+import { TextFieldProps } from './types';
+
+export default function TextField({
+  accessible = true,
+  accessibilityLabel,
+  accessibilityRole,
+  accessibilityState,
+  containerStyle,
+  disabled,
+  error,
+  errorMessage,
+  errorMessageStyle,
+  label,
+  labelStyle,
+  inputStyle,
+  ...rest
+}: TextFieldProps): JSX.Element {
+  const inputContainerStyle: ViewStyle = useMemo(
+    () => ({
+      ...styles.container,
+      ...(disabled && styles.disabled),
+    }),
+    [disabled]
+  );
+
+  return (
+    <View
+      accessible={accessible}
+      accessibilityLabel={accessibilityLabel ?? label}
+      accessibilityRole={accessibilityRole}
+      accessibilityState={{ disabled, ...accessibilityState }}
```

Comment:
Just curious, why are we placing these on the container rather than the input?

Reply:
The label gets announced twice by screen readers otherwise.